### PR TITLE
ADTS-55 Stoping Layout from reload on relationship changes

### PR DESCRIPTION
**Pull request by JoseZarmada**

### CODE REVIEW

**Review by jamescarpinter — APPROVED** [review]

**Review by jamiewilbraham — CHANGES_REQUESTED** [review]
We need to ensure the relationship color is the same when creating new ones

**Comment by JoseZarmada** [comment]
Added the call of the ReColor function to fix the issue

File: client/src/components/GraphViewerComponent/GraphViewerComponent.js [hunk 1]
```diff
@@ -467,10 +467,10 @@ class GraphViewerComponent extends React.Component {
     this.setState({ canShowAllRelationships: true });
   }
 
-  onRelationshipCreate = async relationship => {
+  onRelationshipCreate = relationship => {
     if (relationship) {
       this.cyRef.current.addRelationships([ relationship ]);
-      await this.cyRef.current.doLayout();
+      this.cyRef.current.updateNodeColors();
```
Comment (jamescarpinter) [inline client/src/components/GraphViewerComponent/GraphViewerComponent.js]: Would this make more sense to place inside the addRelationship function itself? Or would that cause other issues?

Keeping the color logic internal to the cytoscape component would be cleaner.

**Review by jamescarpinter — CHANGES_REQUESTED** [review]

File: client/src/components/GraphViewerComponent/GraphViewerComponent.js (hunk 1 above)
Comment (JoseZarmada) [reply client/src/components/GraphViewerComponent/GraphViewerComponent.js]: Done

**Review by jamescarpinter — APPROVED** [review]

**Pull request merged**